### DISCUSSION TITLE
vi build then copy to release directory

### DISCUSSION THREAD

**Discussion by joegittingherdone** [open-watcom/open-watcom-v2#685] [Q&A] [answered]
Hello,
When I try to build vi in bld/nt386.win, I don't see the date of the rel executable viw.exe (inside rel/binnt) change, so I don't think it's being built.  I've tried builder build and I've tried wmake.  What's the correct procedure to only build vi?

**Comment by jmalak** [comment, marked as answer]
OW build system is for developers which works on OW source code.
Therefore it is flexible system and some things is determined by programer for specific needs.
if you run `build` command it only build OW and don't copy binary objects to installation directory tree (release tree).
If you want to create installation directory tree and copy all builded object there you can do it selectively or for full build.
by example if you want to create full OW build and create installation tree then run command `build rel`.
The `OWRELROOT` environment variable control what release tree directory is used. If it is not setup then default `rel` is used.
If you already have OW build and you want create installation tree then run in `bld` directory command `builder cprel`.
If you want create only particular project and update it in installation tree then run in appropriate directory command `builder rel`.
By example if you want only `vi` then run in `bld/vi` command `builder rel`.
If you don't specify `OWRELROOT` environment variable then implicit `rel` subdirectory in OW tree is used or you can specify it on builder command line as `OWRELROOT=<where you want to create or update installation tree>` option (it temporarily overwrite `OWRELROOT` variable for builder command).

**Comment by joegittingherdone** [comment]
> If you want create only particular project and update it in installation tree then run in appropriate directory command builder rel.
By example if you want only vi then run in bld/vi command builder rel.

That works; thank you!
